build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/projects/build.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-env es6 */
+const path = require('path');
+const { FileSystem, Async } = require('@rushstack/node-core-library');
+
+const args = process.argv.slice(2);
+
+const getArgument = (argumentName) => {
+  const index = args.findIndex((e) => e.includes(argumentName));
+  return index >= 0 ? args[index].replace(`${argumentName}=`, '') : undefined;
+};
+
+const shard = getArgument('--shard');
+
+const outputDir = getArgument('--output-directory');
+
+const shardOutputDir = getArgument('--shard-output-directory');
+
+const outputDirectory = shard ? (shardOutputDir ? shardOutputDir : outputDir) : undefined;
+
+async function runAsync() {
+  await Async.sleepAsync(2000);
```

Comment:
Why so long?

Reply:
Adjusted this to help me see if sharding is working as I would expect, made it a little clearer in timeline view and during cobuild logging if it was working.